mlflow/langchain/__init__.py

```diff
@@ -638,6 +640,24 @@ def predict(
 
         return self._predict_with_callbacks(data, params, callback_handlers=callbacks)
 
+    def _update_tracing_prediction_context(self, callback_handlers):
```

Comment:
Can we use a more descriptive name for this method?

Reply:
Filed follow-up https://github.com/mlflow/mlflow/pull/12053